features/cpts_ui/change_role.feature

```diff
@@ -84,6 +84,21 @@ Feature: Users are able to change their roles, if they have multiple roles with
     Then I am on the change role page
     And I can see the role that has been pre selected
 
+  @allure.tms:https://nhsd-jira.digital.nhs.uk/browse/AEA-5046
+  Scenario: User refreshes the page and retains roles with access
+    Given I am logged in as a user with multiple access roles
+    And I have confirmed a role
+    And I am on the change your role page
+    And I click a change role role card
+    When I click the change link next to the role text
+    Then I am on the change role page
+    And I can see the role that has been pre selected  
+
+    When I refresh the page
```

Comment:
@seansteberisal might not like this, it's supposed to be strictly Given -> When -> Then. In this case, I think you're alright to just remove the middle Then stuff, since the banner existing at that point is already done elsewhere. i.e. 
```
Given I'm logged in
When I do all the login stuff
And I refresh the page
Then I see the selected role
```